week1/2_Arithmetics.ipynb

```diff
@@ -4,23 +4,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<img src=\"images/logo.jpg\" style=\"display: block; margin-left: auto; margin-right: auto;\" alt=\"לוגו של מיזם לימוד הפייתון. נחש מצויר בצבעי צהוב וכחול, הנע בין האותיות של שם הקורס: לומדים פייתון. הסלוגן המופיע מעל לשם הקורס הוא מיזם חינמי ללימוד תכנות בעברית.\">"
+    "<img alt=\"לוגו של מיזם לימוד הפייתון. נחש מצויר בצבעי צהוב וכחול, הנע בין האותיות של שם הקורס: לומדים פייתון. הסלוגן המופיע מעל לשם הקורס הוא מיזם חינמי ללימוד תכנות בעברית.\" src=\"images/logo.jpg\" style=\"display: block; margin-left: auto; margin-right: auto;\"/>"
```

Comment:
If it is OK on your side, I'd prefer not to move around attributes in places where it doesn't make a difference :)

Reply:
I used a html parser (BeautifulSoup) to do the reformatting - there's no direct control on all the details of the generated html